README.md

```diff
@@ -76,6 +76,7 @@ Options:
   --datadir, -d  feed data directory, if ommited RAM memory will be used[string]
   --output, -o   log output directory or file, if not provided output will be
                  logged to console.                                     [string]
+  --prefix, -p   path prefix to be omitted                              [string]
```

Comment:
hmm maybe name could be like --remote-prefix and alias -rp or no alias, -p usually is used for paths or ports, would be a bit confusing, wdyt?

Reply:
agree